From e0ec68dccb48c52f21b4681e7ba2d1a44a8b067d Mon Sep 17 00:00:00 2001
From: Sebastian Reichel <sebastian.reichel@collabora.com>
Date: Tue, 21 Apr 2026 18:01:37 +0200
Subject: [PATCH 137/172] usb: typec: fusb302: rename init_gpio into
 fusb302_init_irq

Add the fusb302_ prefix to init_gpio, since it is used by all
the other functions and allows easy figuring out that it is a
local function.

Signed-off-by: Sebastian Reichel <sebastian.reichel@collabora.com>
---
 drivers/usb/typec/tcpm/fusb302.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/usb/typec/tcpm/fusb302.c b/drivers/usb/typec/tcpm/fusb302.c
index 7148f0f3c1f..a28b90b98e4 100644
--- a/drivers/usb/typec/tcpm/fusb302.c
+++ b/drivers/usb/typec/tcpm/fusb302.c
@@ -1644,7 +1644,7 @@ static void fusb302_irq_work(struct work_struct *work)
 	enable_irq(chip->gpio_int_n_irq);
 }
 
-static int init_gpio(struct fusb302_chip *chip)
+static int fusb302_init_irq(struct fusb302_chip *chip)
 {
 	struct device *dev = chip->dev;
 	int ret = 0;
@@ -1759,7 +1759,7 @@ static int fusb302_probe(struct i2c_client *client)
 	if (client->irq) {
 		chip->gpio_int_n_irq = client->irq;
 	} else {
-		ret = init_gpio(chip);
+		ret = fusb302_init_irq(chip);
 		if (ret < 0)
 			return ret;
 	}
-- 
2.53.0

